src/lib/api/patrons/patron.js

```diff
@@ -37,7 +37,7 @@ class QueryBuilder {
     if (!email) {
       throw TypeError('Email argument missing');
     }
-    this.patronQuery.push(`email:${email}${wildcard ? '*' : ''}`);
+    this.patronQuery.push(`email:${email}`);
```

Comment:
I am not 100% sure about this change, the `*` was there for something probably? And this is changing any usage of this query builder, not only `CheckoutSearch`.

Probably the removal of the `*` is good (did you test many different cases?), but wouldn't be better in the checkout search to avoid to specify fields? We are limiting to email and name, but we need to support the person_id too...

Reply:
Actually I have double check, and we are missing the config `PATRONS.customFields` in `cds-ils`, see [here](https://github.com/inveniosoftware/react-invenio-app-ils/blob/e1d10e64ef485c2ddcebd64c6ba4617040080f6e/src/lib/config/defaultConfig.js#L794) and how it is used [here](https://github.com/inveniosoftware/react-invenio-app-ils/blob/e1d10e64ef485c2ddcebd64c6ba4617040080f6e/src/lib/pages/backoffice/Actions/CheckOut/state/actions.js#L38).
Can you add a PR in `cds-ils` to add the config?